README.md

```diff
@@ -1,3 +1,4 @@
+<!-- markdownlint-disable -->
```

Comment:
**Suggestion**
We need to update documentation links in this file. Plus we can remove major update section from top level readme.